ldes-server-domain/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/domain/ldes/eventstream/services/EventStreamConverterImpl.java

```diff
@@ -55,9 +52,17 @@ private List<Statement> addViewStatements(List<TreeNode> views) {
 
 		views.forEach(view -> {
 			Resource viewResource = createResource(view.getFragmentId());
-
 			statements.add(createStatement(viewResource, RDF_SYNTAX_TYPE, createResource(TREE_NODE_RESOURCE)));
-			statements.addAll(relationStatementConverter.getRelationStatements(view.getRelations(), viewResource));
+			view.getRelations()
```

Comment:
can you see if this is covered by a unit test?

Reply:
Correct remark